includes/container/container.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Elementor\Container;
+
+use Exception;
+use ElementorDep\DI\ContainerBuilder;
+use ElementorDep\DI\Container as DIContainer;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor Container.
+ *
+ * Elementor container handler class is responsible for containerization
+ * manager classes and their dependencies.
+ *
+ * @since 3.24.0
+ */
+
+class Container extends ContainerBuilder {
+
+	protected static $instance;
+
+	public function __construct() {
```

Comment:
How do I use it in `elementor-pro` or in another module? is there is a way to `bind` for another module?

Reply:
@Nevoss Elementor-Pro will have its separate installation for the PHP-DI